cmd/rekor-cli/app/validate.go

```diff
@@ -60,13 +63,14 @@ func validateSHA256Value(v string) error {
 		hash = split[1]
 	}
 
-	s := struct {
-		Prefix string `validate:"omitempty,oneof=sha256"`
-		Hash   string `validate:"required,len=64,hexadecimal"`
```

Comment:
qq, i noticed on the other PR, we dropped `required` from `required,email` - is this behavior equivalent with this new library?

Reply:
yes, the check fails if the criteria isn't met